src/logger.py

```diff
@@ -1,9 +1,39 @@
 """Logging configuration for the dune-sync package."""
 
 import logging
+import sys
 from os import getenv
 
-# TODO maybe support custom logging config loaded from the config file
-logging.basicConfig(level=getenv("LOG_LEVEL", "INFO"))
+
+class InfoFilter(logging.Filter):
+    """Filter that only allows records at INFO or WARNING level."""
+
+    def filter(self, record):
+        """Return True if we should log this record."""
+        return record.levelno in (logging.INFO, logging.WARNING)
```

Comment:
Should DEBUG and TRACe also be part of the filter?

Reply:
Good eye. Might be better here to just do the opposite of ERROR so we catch them all.

Reply:
69e7fe6